packages/components/organisms/f-header/stories/header.stories.js

```diff
@@ -44,12 +44,24 @@ export const HeaderComponent = () => ({
         },
         showHelpLink: {
             default: boolean('Show help link', true)
+        },
+        showCountrySelector: {
+            default: boolean('Show country selector', true)
         }
     },
     parameters: {
         notes: 'some documentation here'
     },
-    template: '<vue-header :userInfoProp="userInfoProp" :showOffersLink="showOffersLink" :showHelpLink="showHelpLink" :locale="locale" :headerBackgroundTheme="headerBackgroundTheme" :showDeliveryEnquiry="showDeliveryEnquiry" :showLoginInfo="showLoginInfo" />'
+    template: `
+        <vue-header
+            :userInfoProp="userInfoProp"
+            :showOffersLink="showOffersLink"
+            :showHelpLink="showHelpLink"
+            :locale="locale"
+            :headerBackgroundTheme="headerBackgroundTheme"
+            :showDeliveryEnquiry="showDeliveryEnquiry"
+            :showLoginInfo="showLoginInfo"
+            :showCountrySelector="showCountrySelector" />`
```

Comment:
It might be necessary to put the locale into the `:key` here to force rerender of the component when changing the locale knob, as we're using it in order to generate initial state in `data()`:
```suggestion
            :showCountrySelector="showCountrySelector"
            :key="locale" />`
```